docs/src/content/docs/integrations/typescript/guided-tutorial/cosmos-union-sepolia.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Send Funds Union → Sepolia
+description: Example transfer from Union to Sepolia.
+---
+
+This guide walk's you through writing our example ["Send Funds Union -> Sepolia"](/integrations/typescript/examples/cosmos/send-funds-union-to-sepolia/). The goal of this guide it to show you how to create a token order and submit it to the [UCS03 - ZKGM](/ucs/04) interface.
+
+Relevant imports will be included with each step. Outside of libraries provided by Union, this guide uses [Effect](https://effect.website/).
+
+## Program
+
+This first section is a walk through of creating the program section of the send funds example.
+
+### 1. Program Declaration
+
+Begin by using Effect to create a program function.
+
+```ts
+import { Effect, Logger } from "effect"
+
+const program = Effect.gen(function*() {})
+```
+
+### 2. Source and Destination
+
+Using the `ChainRegistry` from the Union TS SDK, you can declare the source and destination chains used in this example. In this case, the source is Union Testnet (`union.union-testnet-10`) and the destination is Sepolia (`ethereum.11155111`).
+
+```ts
+import { ChainRegistry } from "@unionlabs/sdk/ChainRegistry"
+import { UniversalChainId } from "@unionlabs/sdk/schema/chain"
+import { Effect, Logger } from "effect"
+
+const program = Effect.gen(function*() {
+  const source = yield* ChainRegistry.byUniversalId(
+    UniversalChainId.make("union.union-testnet-10"),
+  )
+  const destination = yield* ChainRegistry.byUniversalId(
+    UniversalChainId.make("ethereum.11155111"),
+  )
+})
+```
+
+:::note
+The ID's provided here are Universal Chain ID's as defined by [UCS04](/ucs/04).
+:::
+
+### 3. Token Order
+
+Now you can define the token order that will be responsible for the transfer. This example uses the `TokenOrderV2` TS interface.
+
+To construct the token order, you will need the token contract/denom on both the source chain (`baseToken`) and the destination chain (`quoteToken`). In this case, we are using the relevant denom and contract address for U.
```

Comment:
```suggestion
To construct the token order, you will need the token contract/denom on both the source chain (`baseToken`) and the destination chain (`quoteToken`). In this case, we are using the relevant denom and contract address for `U`.
```

Reply:
feel free to ignore this one, bit of a stylistic choice whether we want to use `U` or just U